tests/replication/test_federation_sender_shard.py

```diff
@@ -35,7 +35,9 @@ class FederationSenderTestCase(BaseMultiWorkerStreamTestCase):
 
     def default_config(self):
         conf = super().default_config()
-        conf["send_federation"] = False
+        # Setting this to an empty list disables federation sending. This is actually
+        # declared in super(), so this entire override is probably redundant.
+        conf["federation_sender_instances"] = []
```

Comment:
```suggestion
        # Setting this to an empty list disables federation sending. This is
        # redundant with the default configuration.
        conf["federation_sender_instances"] = []
```

Although I'm honestly inclined to remove this block and maybe put a docstring on `FederationSenderTestCase` which says that we update the `federation_sender_instances` config in each test in order to re-enable federation sending (which is disabled by default).

Reply:
Losing it works for me. I'll do that.